### ISSUE TITLE
Performance considerations

### ISSUE THREAD

**Issue by tuffnatty** [barseghyanartur/transliterate#44]
I am calling `translit` on millions of small strings. While profiling, I have noticed that 83% of the time in each call to `translit()` function is spent on instantiating a language pack. After changing my code to:
```python
import transliterate                                                                                                                         
                                                                                                                                             
transliterate.utils.ensure_autodiscover()                                                                                                    
ru_translit = transliterate.base.registry.get('ru')().translit 
```
and then calling the `ru_translit` function instead of `translit(..., 'ru')`, I've achieved a huge performance boost, roughly 5x. Too bad there is no official (documented) API for that.

**Comment by barseghyanartur** [comment]
@tuffnatty:

Actually, a good point. My suggestion would be to make another function called `get_translit_function` (in the package core).

```python
def get_translit_function(language_code):
    """Return translit function for the language given.
    :param str language_code:
    :return callable:
    """
    ensure_autodiscover()

    cls = registry.get(language_code)

    if cls is None:
        raise LanguagePackNotFound(
            _("Language pack for code %s is not found." % language_code)
        )

    language_pack = cls()
    return language_pack.translit
```

You're welcome to make a pull request (if you have time). Otherwise, I'll make it myself.

Thanks for reporting this.

**Comment by tuffnatty** [comment]
@barseghyanartur : Sounds great! However, my immediate problem is solved, and after publicly sharing the solution, not much motivation is left to make a PR ;-( So if you have time please do it.

**Comment by barseghyanartur** [comment]
@tuffnatty:

Yep. No problems. Thanks again!

**Comment by barseghyanartur** [comment]
Implemented in 1.10.2.
